code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -11,11 +12,25 @@ GLOBAL_LIST_INIT(hardcoded_gases, list(/datum/gas/oxygen, /datum/gas/nitrogen, /
 		gas_info[META_GAS_MOLES_VISIBLE] = initial(gas.moles_visible)
 		if(initial(gas.moles_visible) != null)
 			gas_info[META_GAS_OVERLAY] = new /obj/effect/overlay/gas(initial(gas.gas_overlay))
-		gas_info[META_GAS_FUSION_POWER] = initial(gas.dangerous)
+		gas_info[META_GAS_FUSION_POWER] = initial(gas.fusion_power)
 		gas_info[META_GAS_DANGER] = initial(gas.dangerous)
 		gas_info[META_GAS_ID] = initial(gas.id)
 		.[gas_path] = gas_info
 
+/proc/nonreactive_gas_list() //all the gases not included in _any_ reaction
+	var/list/nonreactive_gases = GLOB.meta_gas_info.Copy()
+	for(var/g in GLOB.meta_gas_info)
+		var/datum/gas/gas = g
+		for(var/r in SSair.gas_reactions)
+			if(!gas in nonreactive_gases) //if its been removed already we can move on
+				continue
+			var/datum/gas_reaction/reaction = r
+			if((gas in reaction.min_requirements) || (gas in reaction.max_requirements)) //if the gas is mentioned in the requirements of a reaction
+				nonreactive_gases -= gas
+	. = list()
+	for(var/path in nonreactive_gases)
+		. += new path
```

Comment:
Why new here ? Not that it matters right now. And no easy way to find out the nonreactive chain is coming to mind, maybe start from the air mix as preset. with that as a base it should be easier to expand.